src/Graze/Monolog/Handler/RaygunHandler.php

```diff
@@ -55,23 +65,26 @@ protected function write(array $record)
     /**
      * @param array $record
      */
-    protected function writeError(array $record)
+    protected function writeError(array $record, $tags = null, $customData = null, $timestamp = null)
```

Comment:
If I'm reading the RayGun code right, `$tags` and `$customerData` should be an arrays, so let's type-hint those correctly.